src/tesseract.rs

```diff
@@ -74,10 +74,28 @@ impl TessApi {
         }
     }
 
+    /// Provide an image for Tesseract to recognize.
+    /// 
+    /// set_image clears all recognition results, and sets the rectangle to the full image, so it
+    /// may be followed immediately by a `[Self::get_utf8_text]`, and it will automatically perform
+    /// recognition. 
     pub fn set_image(&mut self, img: &leptonica::Pix) {
+        // "Tesseract takes its own copy of the image, so it need not persist until after Recognize"
         unsafe { capi::TessBaseAPISetImage2(self.raw, img.raw as *mut capi::Pix) }
     }
 
+    // I have no idea if this is correct:
```

Comment:
these will show up in the public documentation for this method, better put it inside the method for internal documentations.

Reply:
As far as I know, Rustdoc only takes comments into accounts that are preceded with `///`. `//` are untouched

Reply:
Oh, yeah, you are right, i didn't expect those to be `//`. That said, I still recommend moving it into the method body so it's easier to manage when we need to add `///` docs.